dev/build_release.py

```diff
@@ -146,6 +111,20 @@ def determine_package_version(platform, gradle_root):
     if re.match('-$', version): version = version + '0'
     return version
 
+def run_shell_and_log(cmd_list, logfile, cwd=None):
+  print 'Executing command list: {}'.format(cmd_list)
```

Comment:
My thought here was to provide some logging to the caller as well as in the individual log files, but I think there's enough information just in the log files to determine what happened. I'll add the command context and output in the log files only.